research/src/sources/primer.json5

```diff
@@ -15,69 +15,84 @@
       "concepts": [
         {
           "name": "primary",
-          "category": "type",
           "image": "primer-primary-button.png",
           "$comment": "For form actions and primary page actions."
         },
         {
           "name": "default",
-          "category": "type",
           "image": "primer-default-button.png",
-          "$comment": "Used to indicate the primary action on a page"
+          "$comment": "Used to indicate the primary action on a page."
+        },
+        {
+          "name": "small",
+          "image": "primer-small-button.png",
+          "$comment": "Used to indicate the primary action on a page."
         },
         {
           "name": "danger",
-          "category": "type",
           "image": "primer-danger-button.png",
-          "$comment": "They help reiterate that the intended action is important or potentially dangerous (e.g., deleting a repo or transferring ownership)"
+          "$comment": "They help reiterate that the intended action is important or potentially dangerous (e.g., deleting a repo or transferring ownership)."
         },
         {
           "name": "outline",
-          "category": "type",
           "image": "primer-outline-button.png",
           "$comment": "Outline buttons downplay an action as they appear like boxy links."
         },
         {
           "name": "link",
-          "category": "type",
           "image": "primer-link-button.png",
-          "$comment": "Styled like a link, but uses a <button> element"
+          "$comment": "Styled like a link, but uses a <button> element."
         },
         {
           "name": "group",
-          "category": "type",
           "image": "primer-button-group.png",
-          "$comment": "Groups of buttons"
+          "$comment": "Groups of buttons."
         },
         {
           "name": "block",
-          "category": "type",
           "image": "primer-block-button.png",
-          "$comment": "Full width button"
+          "$comment": "Full width button."
+        },
+        {
+          "name": "octicon",
+          "openUIName": "icon",
```

Comment:
The `name` key will always capture the original named in the source material.  In this case, the design system called this an "Octicon Button". 

We are playing with this "openUIName" idea right now when we need to specify a name that a design system uses, such as `octicon button`, and also surface the variant along with other commonly named things, like `icon button`.

The automated page layout will show the `octicon` button in the `icon` concept now, but we still have a record that the original `name` was something else.  We'll likely show this differing information in the automated page soon.